src/templates/base/base_ebook.html

```diff
@@ -2,9 +2,10 @@
 
 {% set metadata = {"authors":["rviscomi"]} %}
 
-{% block image_url %}https://almanac.httparchive.org/static/images/home-hero-bg.png{% endblock %}
-{% block image_height %}562{% endblock %}
-{% block image_width %}715{% endblock %}
+{% block page_url %}https://almanac.httparchive.org{{ url_for(request.endpoint, **get_view_args(lang=language.lang_code, year=year)) }}{% endblock %}
```

Comment:
Do we need this? Seems like it's the default one (set in base.html) so no need to override here?

```suggestion
```

Reply:
Good catch, bad copypasta.